src/city.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+from .config import initial_city_balance, representative_portion
+
+
+class city:
+    Def __init__(self, country_name: str, countries_list: list, x: int, y: int):
```

Comment:
wrong. def shoulb be with a small letter and not a capital.

Reply:
Thank you, I will work something out